pom.xml

```diff
@@ -2031,11 +2071,9 @@
                             <compound implementation="com.github.ferstl.maven.pomenforcers.CompoundPedanticEnforcer">
```

Comment:
It is I think

Reply:
I left it because it's error reporting is much better.
Sort POM just reports POM is not sorted which is kinda not supper clear.
But if we want it to go away let me know will remove it.